app.py

```diff
@@ -6,8 +6,9 @@
 
 from api.jobs_api import jobs_api
 from api.skills_api import skills_api
+
 from dal.models import db
-from dal.load_data import load_data_from_externalapi
+from dal.load_dal import load_data_from_externalapi
```

Comment:
such typos are a bit weird in the code because seems like it somehow did not run before u submitted